coderd/prometheusmetrics/aggregator_test.go

```diff
@@ -80,15 +81,14 @@ func TestUpdateMetrics_MetricsDoNotExpire(t *testing.T) {
 	}
 	expected := []*agentproto.Stats_Metric{
 		{Name: "a_counter_one", Type: agentproto.Stats_Metric_COUNTER, Value: 1, Labels: commonLabels},
-		{Name: "b_counter_two", Type: agentproto.Stats_Metric_COUNTER, Value: 4, Labels: commonLabels},
 		{Name: "b_counter_two", Type: agentproto.Stats_Metric_COUNTER, Value: -9, Labels: []*agentproto.Stats_Metric_Label{
 			{Name: "agent_name", Value: testAgentName},
 			{Name: "lizz", Value: "rizz"},
 			{Name: "username", Value: testUsername},
 			{Name: "workspace_name", Value: testWorkspaceName},
 			{Name: "template_name", Value: testTemplateName},
 		}},
-		{Name: "c_gauge_three", Type: agentproto.Stats_Metric_GAUGE, Value: 5, Labels: commonLabels},
+		{Name: "b_counter_two", Type: agentproto.Stats_Metric_COUNTER, Value: 4, Labels: commonLabels},
```

Comment:
would be good to test with some gnarly label values like `|=,`

Reply:
A pipe character is gone since we switched to the struct key, but I added test cases with characters `,` or `=`.